src/Microsoft.AspNetCore.SignalR/HubConnectionContext.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Security.Claims;
+using System.Threading.Tasks.Channels;
+using Microsoft.AspNetCore.SignalR.Internal.Protocol;
+using Microsoft.AspNetCore.Sockets;
+
+namespace Microsoft.AspNetCore.SignalR
+{
+    public class HubConnectionContext
+    {
+        private readonly WritableChannel<byte[]> _output;
+        private readonly ConnectionContext _connectionContext;
+
+        public HubConnectionContext()
```

Comment:
remove - I don't see it being used anywhere and it creates an unusable instance.